examples/ios/swift/Migration/MigrationExample.swift

```diff
@@ -0,0 +1,61 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Foundation
+import RealmSwift
+
+struct MigrationExample {
+
+    func addExampleDataToRealm(_ exampleData: (Realm) -> Void) {
+        let url = realmUrl(for: schemaVersion, usingTemplate: false)
+        let configuration = Realm.Configuration(fileURL: url, schemaVersion: UInt64(schemaVersion))
+        let realm = try! Realm(configuration: configuration)
+
+        try! realm.write {
+            exampleData(realm)
+        }
+
+        // Uncomment the following line to print the location of the newly created Realm.
+//        print("Realm created at: \(String(describing: configuration.fileURL!)).")
```

Comment:
It's unclear to me why this is here– if for some reason the user does need the location of the Realm, this comment should explain why they might, and printing it definitely doesn't seem like the right way to inform the user of this information

Reply:
My initial idea was to make it easier to find the created realm example files. Removing this might make things easier to understand for the user though. 👍